test/code-highlighter.html

```diff
@@ -85,15 +85,33 @@
       it('tapping on last line should not fire', function (done) {
         elem.code = 'one liner \n'
         var icon = Polymer.dom(elem.root).querySelector('iron-icon');
-        flush(function () {
-          elem.addEventListener('add-hunk', function () {
-            done(new Error('This event should not have been fired'));
-          });
+        elem.addEventListener('add-hunk', function () {
+          done(new Error('This event should not have been fired'));
         });
         MockInteractions.tap(icon);
         setTimeout(done, 10);
       });
 
+      it('tapping twice only fires comment', function () {
+        var icon = Polymer.dom(elem.root).querySelector('iron-icon');
+        var countHunk = 0;
+        var countComment = 0;
+
+        elem.addEventListener('add-hunk', function (e) {
+          assert.equal(countHunk, 0);
+          assert.equal(e.detail.code, ' second line \n');
+          countHunk++;
+        });
+        elem.addEventListener('add-comment', function () {
+          assert.equal(countComment, 0);
+          countComment++;
+        });
+
+        MockInteractions.tap(icon);
+        MockInteractions.tap(icon);
+
+      });
```

Comment:
This test does not work as intended. It finished before the two event handlers get the chance to do the assertions. Use `done` to prevent this.